Exam/CT/src/exercice2/MaskedImage.java

```diff
@@ -0,0 +1,44 @@
+package exercice2;
+
+public class MaskedImage {
+	
+	private int [][]pixels;
+	public boolean[][] inclusionMask ;
+	
+	public MaskedImage(int [][] input)
+	{
+		this.pixels = input;
+		inclusionMask = new boolean[pixels.length][pixels.length];
```

Comment:
- pixels is not necessarily square.
- why initialize `inclusionMask` here and not in `initMask`?